### PR TITLE
feat: implement chr, ord, and hex

### PR DESCRIPTION
**Pull request by thesayyn**
### Type of change

- New feature or functionality (change which adds functionality) #418 

- Relevant documentation has been updated

### Test plan

- New test cases added

### CODE REVIEW

**Review by alexeagle — APPROVED** [review]

File: lib/string.bzl [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+
+"Utilities for strings"
```
Comment (gregmagolan) [inline lib/string.bzl:3]: nit: the comment in strings.py is now out of date since it refers to base64.bzl

**Review by gregmagolan — APPROVED** [review]
🌮

**Comment by alexeagle** [comment]
Another nit: how did we decide on `string.bzl` as the public API rather than `strings.bzl` ?

**Comment by thesayyn** [comment]
> Another nit: how did we decide on `string.bzl` as the public API rather than `strings.bzl` ?

we haven't actually. open to suggestions. `strings.bzl` sounds more appropriate to me.

**Comment by thesayyn** [comment]
to get this going; assuming Alex and I agree on `strings.bzl`, should I rename this to strings.bzl?

**Comment by alexeagle** [comment]
yes please, let's go with that. Unless you can find some other starlark implementation that already has a name for a module that provides these (but I expect there isn't since they're proposed as builtin keywords)

**Comment by thesayyn** [comment]
> but I expect there isn't since they're proposed as builtin keywords

true. renamed to strings

**Pull request merged**